### ISSUE TITLE
Forwarded media is not rendered properly

### ISSUE THREAD

**Issue by 6vasia** [majn/telegram-purple#425]
Forwarded non-text messages are rendered as 
> $author wrote: 
> [webpage] or [document] etc

Simple edit like following fixes media render, but messes plain text (text renders twice).
```
--- a/tgp-msg.c
+++ b/tgp-msg.c
@@ -732,7 +732,7 @@ static void tgp_msg_display (struct tgl_state *TLS, struct tgp_msg_loading *C) {
     tgl_peer_t *FP = tgl_peer_get (TLS, M->fwd_from_id);
     
     char *tmp = text;
-    text = tgp_msg_reply_display(TLS, FP, M, "");
+    text = tgp_msg_reply_display(TLS, FP, M, tmp);
     g_free (tmp);
     
     g_return_if_fail(text != NULL);
```

I guess,  tgp_msg_reply_display should be split into reply and forward.

**Comment by Michal-Szczepaniak** [comment]
Any update on that? Its really annoying.

**Comment by powerman** [comment]
Some channels consists mostly of forwarded messages, which makes this plugin nearly useless. Today I had to disable it and run official telegram client instead. This is very sad and inconvenient. Please consider fixing this issue.

**Comment by BenWiederhake** [comment]
Which version are you using?  I thought this was already added in 1.4.1, and works perfectly fine for me, that's why my first guess is an outdated version.

**Comment by BenWiederhake** [comment]
Ah, sorry, I didn't properly read it.  In that case, try out dev-1.4.2, because it already does distinguish between forwards and replies and does stuff with them.  If that doesn't work for you, PRs are very welcome.

**Comment by powerman** [comment]
I'm using latest (8.5 hours ago) master and I get `[webpage]`.

**Comment by BenWiederhake** [comment]
In that case, please try `git checkout dev-1.4.2`, rebuild reinstall and try again.  There are commits in this branch which work on exactly this topic, which is why I'm recommending it.

If that solves your problem, I'll just release 1.4.2 early.

**Comment by powerman** [comment]
At a glance - it helps, thanks!

**Comment by BenWiederhake** [comment]
Should be fixed by [v1.4.2](https://github.com/majn/telegram-purple/releases/tag/v1.4.2).
